docs/guides/milvus/quickstart/prerequisites.md

```
@@ -202,16 +229,16 @@ If you want to keep the namespace and clean up dependencies separately:
 
    For KubeDB-managed etcd, the PVC names follow the pattern `etcd-data-<milvus-name>-etcd-<ordinal>`.
 
-3. Delete MinIO:
+3. Delete the MinIO Tenant:
 
    ```bash
-   $ kubectl delete -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/milvus/quickstart/yamls/minio.yaml
+   $ helm uninstall milvus-minio -n demo
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)